fdbserver/Ratekeeper.actor.cpp

```diff
@@ -657,6 +657,9 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 		reasonID = UID();
 		TraceEvent(SevWarnAlways, "RkSSListFetchTimeout").suppressFor(1.0);
 	}
+	else if(limits->tpsLimit == std::numeric_limits<double>::infinity()) {
+		limits->tpsLimit = SERVER_KNOBS->RATEKEEPER_DEFAULT_LIMIT;
```

Comment:
This might be a stupid question, but does this mean that we can never run more than 1 million tps?